geomstats/information_geometry/dirichlet.py

```diff
@@ -16,11 +16,12 @@
 from geomstats.geometry.base import OpenSet
 from geomstats.geometry.euclidean import Euclidean
 from geomstats.geometry.riemannian_metric import RiemannianMetric
+from geomstats.information_geometry.information_manifold import InformationManifold
 
 N_STEPS = 100
 
 
-class DirichletDistributions(OpenSet):
+class DirichletDistributions(OpenSet, InformationManifold):
```

Comment:
May not be very relevant, but is this the order of inheritance we want? I think the opposite may be more robust. Meaning, if it happens (which hopefully not) we have methods with same name in `OpenSet` and `InformationManifold`, then with the current order, `OpenSet` methods have priority.

Reply:
Done, I also just read that it is recommended to put the Mixins first:
https://dev.to/bikramjeetsingh/write-composable-reusable-python-classes-using-mixins-6lj